plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -469,6 +472,19 @@ public IcebergConfig setSplitManagerThreads(int splitManagerThreads)
         return this;
     }
 
+    public List<String> getAllowedExtraProperties()
+    {
+        return allowedExtraProperties;
+    }
+
+    @Config("iceberg.allowed-extra-properties")
+    @ConfigDescription("List of extra properties that are allowed to be set on Iceberg tables")
+    public IcebergConfig setAllowedExtraProperties(List<String> allowedExtraProperties)
```

Comment:
Do we want to ensure that size of `allowedExtraProperties` is 1 when `*` is specified?